UserTracker.py

```diff
@@ -25,18 +28,40 @@ def get_user_list_string():
 
     return result_string
 
+###
+# Solo Q User Functions
+###
 def get_soloq_user_count():
     return len(bot_soloq_user_set)
 
 def add_soloq_user(user):
     bot_soloq_user_set.add(user)
 
+def is_soloq_user(user):
+    return user in bot_soloq_user_set
+
+def remove_soloq_user(user):
```

Comment:
Currently there's no way to know if this operation actually succeeded. 

Maybe want to add something like that for debugging in future